blackbelt/slack.py

```diff
@@ -12,10 +12,14 @@ def __init__(self, token=None):
 
         self.token = token
 
+    def get_user_id(self):
+        return self.slack.auth.test().body['user_id']
+
     def post_message(self, message, room):
-        return self.slack.chat.post_message(room, message)
+        return self.slack.chat.post_message(room, message, username = "Black Belt", icon_emoji = ":black_joker:")
```

Comment:
We don't have logo. We can ping @natalievenuto or @jandvorak if they create some for Black Belt.